openwisp_controller/config/api/serializers.py

```diff
@@ -48,6 +48,14 @@ class Meta(BaseMeta):
             'modified',
         ]
 
+    def validate_organization(self, value):
+        user = self.context['request'].user
+        if user.is_superuser is False and value is None:
+            raise serializers.ValidationError(
+                'Shared Template can only be created by Admin'
```

Comment:
Shared templates can only be created by superusers.

Flag this string as translatable! I am repeating this over and over with everyone, please remember! Double check any string which is returned by the API or shown in the browsable API and ensure it's flagged as translatable.

Please help me to avoid wasting my time pointing out to same trivial mistakes like this all the time so we can use this time to help you improve more important areas of your work.

Reply:
Sure, I'll take care of this from next time.